ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -1701,6 +1701,27 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission
+- as: gcp-ipi-xpn-minimal-permission-byo-hosted-zone-amd-f28-destructive
```

Comment:
The job name exceed the limit - 61 chars, so suggest - `gcp-ipi-xpn-min-perm-byo-hosted-zone-amd-f28-destructive`.